libraries/wallet/wallet.cpp

```diff
@@ -770,6 +773,14 @@ class wallet_api_impl
 
       return true;
    }
+
+    void quit()
+    {
+        ilog( "Exiting Wallet ..." );
+
+        _Exit(0);
```

Comment:
Like others have commented, calling exit() (or any of its variants) is bad style.

Reply:
yes we told about that a lot ! I will do soon ! Just need to find the best solution and approach.